test/get-property-id.test.js

```diff
@@ -0,0 +1,117 @@
+/* @flow */
+/* global it describe beforeEach afterAll afterEach expect jest */
+import { fetchContainerSettings } from '../src/lib/get-property-id';
+import { getPropertyId, getValidContainer, setContainer } from '../src/lib/local-storage';
+
+import { mockContainer1 } from './mocks';
+
+global.fetch = jest.fn().mockImplementation(async () => {
+  return {
+    status: 200,
+    json: () => mockContainer1
+  };
+});
+
+jest.mock('@paypal/sdk-client/src', () => {
+  return { getMerchantID: jest.fn().mockImplementation(() => [ mockContainer1.owner_id ]) };
+});
+
+describe('get-property-id', () => {
+  afterEach(() => {
+    window.localStorage.clear();
+  });
+
+  afterAll(() => {
+    jest.restoreAllMocks();
+  });
+
+  describe('fetchContainerSettings', () => {
+    beforeEach(() => {
+      window.localStorage.clear();
+    });
+
+    it('requests a container using merchantId and browser location', async (done) => {
+      const expectedUrl = 'https://www.paypal.com/tagmanager/containers/xo?mrid=759SBALRW3ZTY&url=http%3A%2F%2Flocalhost';
+      await fetchContainerSettings({});
+
+      expect(global.fetch).toHaveBeenCalledTimes(1);
```

Comment:
do you need this, if you're testing `toHaveBeenCalledWith` below?

Reply:
Just double-checked. It's necessary: "toHaveBeenCalledWith" doesn't care about how many times a method is called.